### DISCUSSION TITLE
[JS] Create New Records With Upload file

### DISCUSSION THREAD

**Discussion by baguetteism** [pocketbase/pocketbase#1837] [Q&A] [answered]
Can any one give me an example? I have try to create new records upload file many times (Server-Side) and i got "Failed to load the submitted data due to invalid formatting." error from API. 
Thanks.

**Comment by ganigeorgiev** [comment, marked as answer]
This will vary based on how you are reading the request body and sending the files.

Could you provide a code sample how you handling the file upload?

In general, the only requirement PocketBase has is to be a `multipart/form-data` request, aka. if you are using the JS SDK you have to pass a [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData) object as body to the create/update methods.

**Reply by baguetteism** [reply]
Thanks. My problem is solved Solution from #1201
